addons/website_sale/tests/test_customize.py

```diff
@@ -6,13 +6,17 @@
 @odoo.tests.common.tagged('post_install', '-at_install')
 class TestUi(odoo.tests.HttpCase):
 
-    def test_01_admin_shop_customize_tour(self):
+    def setUpListView(self):
```

Comment:
are you trying to make a setUp?

Reply:
Yes/no, it's a set up, but only for test_01 and test _06. The current name is ambiguous?

Reply:
why not use a real setUp in this case. 
you will have a savepoint so creation will be only done once !

Reply:
This setup is not really costly so it's ok to run it twice, but the reasoning was that it was only used in 2 / 6 tests so why make it global.

But I'm ok either way, I'll try with a real setUp, if tests are green too it's fine by me.

Reply:
Ok done, I had to slighly adapt the other tests because some of them used the same product name.